Pytorch 1.12.0 issue

A kind reminder: do not use pytorch 1.12.0 for this project, which would **throw an error**:

_assert not step_t.is_cuda, "If capturable=False, state_steps should not be CUDA tensors." AssertionError: If capturable=False, state_steps should not be CUDA tensors._

I tried with torch 1.11.0, then it works.

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-torch>=1.8
+torch==1.11.0
 torchvision>=0.9.1
 imageio
 imageio-ffmpeg

